komga/src/main/kotlin/org/gotson/komga/domain/service/SeriesLifecycle.kt

```diff
@@ -264,5 +313,5 @@ class SeriesLifecycle(
     }
   }
 
-  private fun ThumbnailSeries.exists(): Boolean = Files.exists(Paths.get(url.toURI()))
+  private fun ThumbnailSeries.exists(): Boolean = url?.toURI()?.let { Files.exists(Paths.get(it)) } ?: false
```

Comment:
If there's no url why return false?

Reply:
If there is no url wouldn't that mean that it doesn't exist no? In others words false

Reply:
But there can be a thumbnail as a bytearray